gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Copied from kotlinc
 org.gradle.jvmargs=-Duser.country=US -Dkotlin.daemon.jvm.options=-Xmx2200m -Dfile.encoding=UTF-8
 
-kotlinBaseVersion=1.7.0-dev-3025
+kotlinBaseVersion=1.7.0-Beta
```

Comment:
Hm, even beta has version numbers. Does it pick the latest one?

Reply:
I'm releasing for this release: https://github.com/JetBrains/kotlin/releases/tag/v1.7.0-Beta it does seems to not have version numbers other than just beta.

link to maven artrifact: https://search.maven.org/artifact/org.jetbrains.kotlin/kotlin-gradle-plugin/1.7.0-Beta/jar

Reply:
Interesting. Beta versions I saw are here: https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies/org/jetbrains/kotlin/analysis-api-standalone-for-ide/

Reply:
that one is some other JetBrains managed dev repositories, the versioning is quite confusing tbh